components/sendgrid/actions/list-blocks/list-blocks.mjs

```diff
@@ -0,0 +1,71 @@
+import validate from "validate.js";
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  key: "sendgrid-list-blocks",
+  name: "List Blocks",
+  description: "Allows you to list all email addresses that are currently on your blocks list. [See the docs here](https://docs.sendgrid.com/api-reference/blocks-api/retrieve-all-blocks)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    startTime: {
+      type: "integer",
+      label: "Start Time",
+      description: "The start of the time range when a blocked email was created (inclusive). This is a unix timestamp.",
+      optional: true,
+    },
+    endTime: {
```

Comment:
I notice some repeatable uses of this variable through the actions. Maybe we can add it into `app`?